### DISCUSSION TITLE
Vertical text

### DISCUSSION THREAD

**Discussion by ther12k** [JaidedAI/EasyOCR#705] [Q&A]
If I want to recognize vertical text like this :

E
A
S
Y
O
C
R

what model language need to be used? or it needs to be custom trained?

**Comment by faisnain** [comment]
> If I want to recognize vertical text like this :
> 
> E A S Y O C R
> 
> what model language need to be used? or it needs to be custom trained?

Hi.. Did you find solution to this? If yes How did you handle it?
